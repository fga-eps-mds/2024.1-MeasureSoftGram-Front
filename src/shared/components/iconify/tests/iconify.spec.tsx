import React from 'react';
import { render } from '@testing-library/react';
import Iconify from '../iconify';

describe('Iconify', () => {
  // Teste de Renderização
  it('should render correctly', () => {
    const { container } = render(<Iconify icon="home" />);
    expect(container.firstChild).toBeTruthy();
  });

  // Teste de Snapshot
  it('should match snapshot', () => {
    const { asFragment } = render(<Iconify icon="home" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
