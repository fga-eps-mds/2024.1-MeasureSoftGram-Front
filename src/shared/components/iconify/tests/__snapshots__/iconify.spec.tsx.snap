// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Iconify should match snapshot 1`] = `
<DocumentFragment>
  <span />
</DocumentFragment>
`;
